### PR TITLE
chore: update stackable-operator version

### PR DESCRIPTION
**Pull request by xeniape**
# Description

Part of https://github.com/stackabletech/issues/issues/601, https://github.com/stackabletech/issues/issues/557

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [x] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.77.2"
+channel = "1.79.0"
```
Comment (NickLarsenNZ) [inline rust-toolchain.toml:2]: Just linking this here (but you would have seen it on the other PR), so feel free to resolve this comment when you read it.

https://github.com/stackabletech/operator-templating/pull/406

**Review by NickLarsenNZ — DISMISSED** [review]
LGTM

Edit: There are errors: https://github.com/stackabletech/secret-operator/actions/runs/9876524815/job/27275842664?pr=467#step:7:671

File: rust-toolchain.toml (hunk 1 above)
Comment (xeniape) [reply rust-toolchain.toml:2]: Yes, was going to look at the errors today :+1: and thanks for the rust-toolchain PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ All notable changes to this project will be documented in this file.
   in the Helm values to v4.0.1 ([#440]).
 - Update the image docker.stackable.tech/k8s/sig-storage/csi-node-driver-registrar
   in the Helm values to v2.10.1 ([#440]).
+- Bumped `stackable-operator` from `0.69.3` to `0.70.0` and other dependencies([#467]).
```
Comment (NickLarsenNZ) [inline CHANGELOG.md:23]: nit

```suggestion
- Bump `stackable-operator` to `0.70.0`, and other dependencies ([#467]).
```

**Review by NickLarsenNZ — DISMISSED** [review]

**Comment by xeniape** [comment]
Passing nightly tests
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/secret-operator-it-custom/11/console

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**